platform/linux-generic/include/odp/api/std_types.h

```diff
@@ -29,6 +29,8 @@ extern "C" {
 
 typedef int odp_bool_t;
 
+typedef uint16_t odp_percent_t;
```

Comment:
In my opinion, let's just push it into `odp/api/spec/std_types.h`.

Reply:
+1 moving will also easy to support ABI compat mode.